<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>create_25percent_file_2</class>
 <widget class="QDialog" name="create_25percent_file_2">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>348</width>
    <height>193</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create % 25  File</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../civiltools.qrc">
    <normaloff>:/shearwall/images/create_25percent_file.svg</normaloff>:/shearwall/images/create_25percent_file.svg</iconset>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Modifiers</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="modifier_spinbox">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
       <property name="value">
        <double>0.010000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="dynamic_analysis_groupbox">
     <property name="text">
      <string>Scale Dynamic</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="create_file">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../civiltools.qrc">
       <normaloff>:/shearwall/images/create_25percent_file.svg</normaloff>:/shearwall/images/create_25percent_file.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../civiltools.qrc"/>
 </resources>
 <connections/>
</ui>
